.travis.yml

```diff
@@ -9,7 +9,14 @@ install:
 - bash .travis-fork-fix
 - go get github.com/mattn/goveralls
 - go get github.com/wadey/gocovmerge
-- curl https://glide.sh/get | sh
+- if [[ $(uname -m) == 's390x' ]]; then
+      wget https://github.com/Masterminds/glide/releases/download/v0.13.1/glide-v0.13.1-linux-s390x.tar.gz;
+      tar -xzf glide-v0.13.1-linux-s390x.tar.gz linux-s390x/glide --strip=1;
+      export PATH=$PWD:$PATH;
+      glide --version;
```

Comment:
What does this call to `glide --version` accomplish?

Reply:
`glide --version` is not required here, will remove it and commit the change.